repo_resource/check.py

```diff
@@ -18,6 +18,8 @@
 
 from repo_resource import common
 
+DEFAULT_CHECK_JOBS = 2
```

Comment:
There is no need for this. Instead, we should initialize:

```
    check_jobs: int = 2
```

In `class SourceConfiguration(NamedTuple)`

Having 2 variables for a single thing is error prone. What happens if we update this in the future?